pystac/utils.py

```diff
@@ -38,19 +46,25 @@ def safe_urlparse(href: str) -> URLParseResult:
 
 
 class StringEnum(str, Enum):
-    """Base Enum class for string enums that will serialize as the string value."""
+    """Base :class:`enum.Enum` class for string enums that will serialize as the string
+    value."""
 
     def __str__(self) -> str:
         return cast(str, self.value)
 
 
 class JoinType(StringEnum):
-    """Allowed join types for the :func:`_join` function."""
+    """Allowed join types for :func:`~pystac.utils.join_path_or_url`."""
 
     @staticmethod
     def from_parsed_uri(parsed_uri: URLParseResult) -> "JoinType":
         """Determines the appropriate join type based on the scheme of the parsed
-        result."""
+        result.
+
+        Args:
+            parsed_uri (urllib.parse.ParseResult) : A named tuple representing the
+                parsed URI.
```

Comment:
Should have a `Returns:` section to match the rest of the updates.

Reply:
Good catch, added in b47dceb3da0952cd4148e710db98318ca9b17c73